kano_settings/system/audio.py

```diff
@@ -99,13 +99,13 @@ def is_HDMI():
 
     if amixer_string.find(hdmi_string) != -1:
         # Make sure config file is up to date
-        if get_setting('Audio') != 'HDMI':
-            set_setting('Audio', "HDMI")
+        if get_setting('Audio') != _("HDMI"):
+            set_setting('Audio', _("HDMI"))
```

Comment:
Was just about to say the same thing, yes. The problem that comes from committing translated strings to config files is that if you ever switch to English, you'll still see the translated strings as the existing setting. Not to mention system wide implications, if there are any obscure ones..